packages/client/lib/utils/sort-peers.js

```diff
@@ -0,0 +1,10 @@
+const orderBy = require('lodash.orderby')
+
+/**
+ * Sorts the peers, in place, by block height and delay
+ */
+const sortPeers = peers => {
```

Comment:
As the const is only used once, could we directly assign it to `module.exports`?